iothub/device/src/Common/Singleton.cs

```diff
@@ -39,7 +40,7 @@ public Task OpenAsync(TimeSpan timeout, CancellationToken cancellationToken)
             return GetOrCreateAsync(timeout, cancellationToken);
         }
 
-        public Task CloseAsync(CancellationToken cancellationToken)
+        public Task CloseAsync()
```

Comment:
Why not remove this impl?

Reply:
Even though this really isn't async all the way down it seems better to be consistent we have OpenAsync, GetOrCreateAsync, etc..